packages/maskbook/src/plugins/Wallet/services/assets.ts

```diff
@@ -39,46 +39,29 @@ export async function getAssetsListNFT(
     provider: CollectibleProvider,
     page?: number,
     size?: number,
-) {
+): Promise<{ assets: ERC721TokenDetailed[]; hasNextPage: boolean }> {
     if (provider === CollectibleProvider.OPENSEAN) {
         const { assets } = await OpenSeaAPI.getAssetsList(address, { chainId, page, size })
         return {
             assets: assets
                 .filter((x) => ['ERC721', 'ERC1155'].includes(x.asset_contract.schema_name))
```

Comment:
It should filter ERC1155 tokens out if ERC1155 gave up.

Reply:
oh done